operational_monitoring/operational_monitoring.model.lkml

```diff
@@ -1,6 +1,6 @@
 connection: "telemetry"
 label: "Operational Monitoring"
-include: "//looker-hub/operational_monitoring/explores/*"
-include: "//looker-hub/operational_monitoring/dashboards/*"
+include: "//looker-hub/operational_monitoring/explores/[!0-9]*"
+include: "//looker-hub/operational_monitoring/dashboards/[!0-9]*"
```

Comment:
How did you figure out that this `[!0-9]` syntax works?  [Looker's docs for `include`](https://cloud.google.com/looker/docs/reference/param-model-include) don't seem to mention it (only the `*` and `**` wildcard characters).

Reply:
I saw this post and figured I'll try it with digits: https://www.googlecloudcommunity.com/gc/Technical-Tips-Tricks/Can-I-exclude-views-instead-of-include-them/ta-p/592271